cmd/kubeadm/app/util/config/upgradeconfiguration.go

```diff
@@ -38,17 +38,6 @@ import (
 
 var componentCfgGV = sets.New(kubeproxyconfig.GroupName, kubeletconfig.GroupName)
 
-// DefaultUpgradeConfiguration return a default UpgradeConfiguration
```

Comment:
some of the refactors of this file is to make it look like other xxconfigruation.go files in the same package.